### PR TITLE
CI: Update kubernetes to 1.7.8 and execute test without a tty

### LINKED ISSUE #563: Update k8s version used for testing to 1.7.5
we are still using version 1.6.7, with the update of CRI-O last week, we are now compatible with latest k8s

### PR DESCRIPTION
**Pull request by chavafg**
Updates kubernetes version used in the tests from 1.6.7 to
1.7.8. 

Jenkins job is not able to open a tty when running this test.
As this test is not required to use a tty, removing -t option
from the kubectl command.

Fixes #563.

Signed-off-by: Salvador Fuentes <salvador.fuentes@intel.com>

### CODE REVIEW

**Comment by MarioCarrilloA** [comment]
lgtm

**Comment by grahamwhaley** [comment]
@chavafg Missing a Fixes (caught by checkcommits)
other than that,
lgtm

**Comment by devimc** [comment]
lgtm

**Comment by devimc** [comment]
lgtm

**Comment by mcastelino** [comment]
@chavafg can you also update https://github.com/clearcontainers/runtime/wiki/Clear-Containers-and-Kubernetes to use this script as a way to perform the setup.

File: integration/kubernetes/nginx.bats [hunk 1]
```diff
@@ -30,22 +30,21 @@ setup() {
...
 	sudo -E kubectl expose deployment "$service_name" --port=80
 	sudo -E kubectl get svc,pod
 	# Wait for nginx service to come up
 	waitForProcess "$wait_time" "$sleep_time" "$cmd"
-	run bash -c "sudo -E kubectl run test-nginx --rm -ti --restart=Never \
-		--image=$busybox_image -- wget --timeout=1 $service_name > $output_file"
-	[ "$status" -eq 0 ]
-	grep "index.html" "$output_file"
+	busybox_pod="test-nginx"
+	sudo -E kubectl run $busybox_pod -i --image="$busybox_image" \
```
Comment (grahamwhaley) [inline integration/kubernetes/nginx.bats]: just to note - this patch seems to do somewhat more than just remove the 't' flag :-)

**Comment by chavafg** [comment]
Seems like Openshift is not happy with this change :(  
Investigating more...

**Pull request merged**